### ISSUE TITLE
Model mismatch, could not reproduce results

### ISSUE THREAD

**Issue by pradipto111** [cool-xuan/Imbalanced_SAM#1]
I face the following issues while using this repository:

1. The claimed results for CIFAR100-LT were using the ResNet32 dataset. However, the given code uses the ResNet34 model.
2. Using the out-of-the-box code on CIFAR100-LT yields an overall 42.5 % accuracy (README claims 51.1).
3. After manually integrating the ResNet32 model and training ImbSAM, I got an overall accuracy of 38.6%.

Kindly help me address these concerns. Thanks!

**Comment by cool-xuan** [comment]
Thanks for your recognition of our work. I try to address your concerns as follows:
1. **Model architecture**: 
  The backbone we used follows the GitHub project of [LTR-weight-balancing](https://github.com/ShadeAlsha/LTR-weight-balancing). The main difference between our ResNet-32 and ResNet-34 exists in the first conv: Conv(k=3, s=1, p=1) v.s. Conv(k=7, s=2, p=3). In this setting, the extracted feature has a 2x bigger resolution.
2. **Reproduction**: 
  We have uploaded our running environment in `environment.yml', where the version of pytorch may play a crucial role:
    ```
    python=3.7
    pytorch=1.7.1
    cuda=11.0
    torchvision=0.8.2
    ```
    Moreover, we also check the uploaded code and make some updates. Please re-clone our repo and create the environment following the uploaded `environment.yml'.
